pkgs/development/python-modules/cruft/default.nix

```diff
@@ -0,0 +1,64 @@
+{ buildPythonPackage
+, cookiecutter
+, fetchFromGitHub
+, gitpython
+, lib
+, poetry-core
+, pytestCheckHook
+, pytest_7
+, typer
+}:
+buildPythonPackage rec {
+  pname = "cruft";
+  version = "2.15.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "cruft";
+    repo = "cruft";
+    rev = version;
+    hash = "sha256-qIVyNMoI3LsoOV/6XPa60Y1vTRvkezesF7wF9WVSLGk=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  nativeCheckInputs = [
+    (pytestCheckHook.override { pytest = pytest_7; })
+  ];
+
+  propagatedBuildInputs = [
+    cookiecutter
+    gitpython
+    typer
+  ];
+
+  # Unfortunately, some tests require internet access to fully clone
+  # https://github.com/cruft/cookiecutter-test (including all branches)
+  # which is possible to package, but is annoying and may be not always pure
+  #
+  # see https://discourse.nixos.org/t/keep-git-folder-in-when-fetching-a-git-repo/8590/6
+  #
+  # There are only 13 tests which do not require internet access on moment of the writing.
+  # But some tests are better than none, right?
+  disabledTests = [
+    "test_get_diff_with_add"
+    "test_get_diff_with_delete"
+    "test_get_diff_with_unicode"
+  ];
+
+  disabledTestPaths = [
+    "tests/test_api.py" # only 2 tests pass, and 24 fail. I am going to ignore entire file
+    "tests/test_cli.py"
+  ];
+
+  meta = {
+    changelog = "https://github.com/cruft/cruft/blob/${version}/CHANGELOG.md";
+    description = "Allows you to maintain all the necessary cruft for packaging and building projects separate from the code you intentionally write. Built on-top of, and fully compatible with, CookieCutter.";
```

Comment:
Prefer short, single-sentence descriptions without final periods. 

Descriptions should ideally make sense in a sentence like "(package name) is a (description).", without going on too long describing what it is.

Reply:
Changed to description from project's README

Reply:
Doesn't matter what upstream preffers in an `nixpkgs` description. Please see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes
(Use capitalized beginning, do not start with the package name)

Reply:
Fixed, thanks!